src/controllers/expense.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpense,
+  deleteExpense,
+  createNewExpense,
+  updateExpense,
+} = require('../services/expenses');
+
+const getAllExpensesController = (req, res) => {
+  const { userId, category, from, to } = req.query;
+
+  if (!category || !userId || !from || !to) {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const findExpenses = getAllExpenses(
+    userId,
+    from,
+    to,
+    category
+  );
+
+  if (!findExpenses.length) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(findExpenses);
+};
+
+const getExpenseController = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (!getExpense(expenseId)) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(getExpense(expenseId));
```

Comment:
You've called this function twice, just save it's result to constant and use this value